addons/bestja_project_hierarchy/models.py

```diff
@@ -155,6 +155,14 @@ class Project(models.Model):
         related='organization.level'
     )
 
+    _sql_constraints = [
+        (
+            'invitation_parent_project_organization_uniq',
+            'unique(parent, organization)',
+            "Organizacja może zostać zaproszona do danego projektu przez organizację nadrzedną tylko raz!"
```

Comment:
The commit message should also be updated. Also the first (and only) line consists of 114 columns, which is too long. I suggest adding a linebreak after the first sentence.